app/src/components/ContributionDetail.vue

```diff
@@ -0,0 +1,106 @@
+<template>
+  <template v-if="contributions.length > 0">
+    <section v-for="contribution in contributions" :key="contribution.grantName" class="px-4 md:px-12">
+      <div class="py-4 md:py-6 lg:py-8 border-b border-grey-100">
+        <div class="grid grid-cols-12 gap-x-8 gap-y-4 items-center">
+          <!-- grand image + grant description + optional grant round -->
+          <article class="col-span-12 md:col-span-6 lg:col-span-5">
+            <!-- subgrid -->
+            <div class="grid grid-cols-4 items-center gap-x-4">
+              <!-- img -->
+              <div class="hidden lg:block col-span-1">
+                <!-- do not forget to link the figure to the grant ^^ -->
+                <figure class="aspect-w-16 aspect-h-9 shadow-light cursor-pointer">
+                  <img
+                    class="w-full h-full object-center object-cover group-hover:opacity-90"
+                    :src="contribution.grantLogoURI ? contribution.grantLogoURI : '/placeholder_grant.svg'"
+                  />
+                </figure>
+              </div>
+              <div class="col-span-4 lg:col-span-3" v-if="contribution.grantName && contribution.roundName">
```

Comment:
Nit, but readability it's preferred to put special Vue attributes first, and things like classes last: https://v3.vuejs.org/style-guide/#element-attribute-order-recommended

Reply:
love the detail. Updated in push

Reply:
Also moved the v-else to the front in 9d03355